src/commcare_cloud/fab/deploy_diff.py

```diff
@@ -82,6 +82,32 @@ def get_commit_date(commit):
                 "error": True
             }
 
+    def get_maintenance_pr_context(self):
+        from github import Github
+
+        now = datetime.now().strftime( '%Y-%m-%d')
+        sixweeks_ago = (datetime.now() - timedelta(days=6*7)).strftime( '%Y-%m-%d')
+        created = f"{sixweeks_ago}..{now}"
```

Comment:
nit: slightly simpler:
```suggestion
        created = f">{sixweeks_ago}"
```

Reply:
True!